### PR TITLE
Add logic to convert paths to absolute paths when needed

### LINKED ISSUE #17: Can not use relative paths in volume mappings with podman
```
ansible-playbook qpc_playbook.yml -vv -e install_offline=true -e server_version=0.9.1 -e cli_version=0.9.1 -e server_install_dir=cody 

TASK [server_podman : Start Quipucords server Podman container linked to PostgreSQL container] ***********************************************************************************************************************************************
task path: /qpc_tools/install/roles/server_podman/tasks/start_quipucords_container.yml:55
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "podman run --name quipucords -p 9443:443 -d -e USE_SUPERVISORD=true -e QPC_SERVER_TIMEOUT=120 -e QPC_DBMS_USER=postgres -e QPC_DBMS_PASSWORD=password -e ANSIBLE_LOG_LEVEL=0 -e NETWORK_CONNECT_JOB_TIMEOUT=600 -e NETWORK_INSPECT_JOB_TIMEOUT=10800 -e QPC_SERVER_USERNAME=admin -e QPC_SERVER_USER_EMAIL=admin@example.com -e QPC_SERVER_PASSWORD=qpcpassw0rd -v cody/server/volumes/sshkeys/:/sshkeys:z -v cody/server/volumes/data/:/var/data:z -v cody/server/volumes/log/:/var/log:z -i quipucords:0.9.1", "delta": "0:00:00.081378", "end": "2019-09-26 13:58:46.982771", "msg": "non-zero return code", "rc": 125, "start": "2019-09-26 13:58:46.901393", "stderr": "Error: error creating named volume \"cody/server/volumes/data/\": error running volume create option: names must match [a-zA-Z0-9][a-zA-Z0-9_.-]*: invalid argument", "stderr_lines": ["Error: error creating named volume \"cody/server/volumes/data/\": error running volume create option: names must match [a-zA-Z0-9][a-zA-Z0-9_.-]*: invalid argument"], "stdout": "", "stdout_lines": []}
...ignoring

[root@vrhel7 install]# ls
cody  packages  qpc_playbook.yml  qpc-tools  roles
```
So, if you pass in a relative path for the `server_install_dir`, the directory will be created with the correct subdirectory infrastructure at the correct relative path location and the files will be correctly passed over. However, podman does not allow you to pass in relative paths in the volume mappings. So you can't do:
```
-v relative_path:absolute_path
```

---

Convert any relative paths passed in for `server_install_dir` are converted to absolute paths.

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #17

### CODE REVIEW

File: qpc_tools/cli/install.py [hunk 1]
```diff
@@ -69,6 +69,7 @@ def _validate_args(self):
                 print(_(messages.CLI_INSTALL_MUST_SPECIFY_PORT_AND_HOST))
                 sys.exit(1)
             self.args.configure_server = 'true'
+        validate_and_update_paths(self.args)
```
Comment (myersCody) [inline qpc_tools/cli/install.py]: I am not a huge fan of this name, it seems to general. What do you think about `require_absolute_paths`

File: qpc_tools/utils.py [hunk 2]
```diff
@@ -86,7 +91,12 @@ def log_args(args):
 
 
 def create_ansible_command(namespace_args, playbook):
-    """Build Ansible Command."""
+    """Build Ansible Command.
+
+    :param namespace_args: arguments passed in by the user
+    :param playbook: name of the playbook
+    :returns:  The ansible command that will be run
```
Comment (myersCody) [inline qpc_tools/utils.py:98]: Ooops Thanks for filling out the doc string here.

File: qpc_tools/cli/install.py (hunk 1 above)
Comment (kholdaway) [reply qpc_tools/cli/install.py]: 🌮

Comment (kholdaway) [reply qpc_tools/cli/install.py]: Changed to what we discussed

File: qpc_tools/utils.py (hunk 2 above)
Comment (kholdaway) [reply qpc_tools/utils.py:98]: 😄

**Review by myersCody — APPROVED** [review]

**Pull request merged**